src/libs/ReportUtils.ts

```diff
@@ -1123,6 +1123,7 @@ function getReport(reportID: string, reports: SearchReport[] | OnyxCollection<Re
 
 /**
  * Returns the report
+ * @deprecated Get the data straight from Onyx
```

Comment:
Just curious, why is `getReportNameValuePairs` getting deprecated in this PR? it doesn't seeem related from what I've seen

Reply:
Oh now i see the issue title that this PR is related to, is working on removing `getReportNameValuePairs` :D

Reply:
Ah yes, sorry I did not explain that. You got it though!